src/Symfony/Contracts/Service/ServiceLocatorTrait.php

```diff
@@ -50,10 +50,8 @@ public function has(string $id)
 
     /**
      * {@inheritdoc}
-     *
-     * @return mixed
      */
-    public function get(string $id)
+    public function get(string $id): mixed
```

Comment:
`has` could also add a return type can't it?

Reply:
In the end yes, but as I described in the PR description, DebugClassLoader should do it, so we now need to figure out why it didn't add it on has().